osl-image-prod.yaml

```diff
@@ -0,0 +1,64 @@
+- schema_version: 1
+  name: "operator-builder"
+  version: "1.29.0"
+  from: "registry-proxy.engineering.redhat.com/rh-osbs/rhel8-go-toolset:1.19.4-7"
```

Comment:
I've just checked and the official release of go-toolset is out. There are both 1.19.6 and 1.19.9 versions available.

So you can use either
`from: "registry.access.redhat.com/ubi8/go-toolset:1.19.6"`
or
`from: "registry.access.redhat.com/ubi8/go-toolset:1.19.9"`

Reply:
+1 from me on using the latest.